### ISSUE TITLE
annotation update to 5.3 -genomad

### ISSUE THREAD

**Issue by aclum** [microbiomedata/mg_annotation#36]
In October JGI is updating to annotation 5.3
to sync with that we need to add a WLD for genomad that gets run as part of annotation pipeline.

Recommend setting this up as a separate WDL, having that WDL be imported to full_annotation.wdl

WDL task to wrap genomad.sh
https://code.jgi.doe.gov/img/img-pipelines/img-annotation-pipeline/-/blob/5.3/bin/genomad/genomad.sh?ref_type=heads

Program and DB file will need  to be added to the info file.

There should be a boolean flag to execute this.

**Comment by ssarrafan** [comment]
@aclum @kaijli doesn't appear to be actively worked on.  Can this be backlogged?  Or moved to next sprint?

**Comment by aclum** [comment]
Kaitlyn is out until the 14, backlogging we address missing key issues  in the handoff b/w workflows and nmdc_automation, https://github.com/microbiomedata/issues/issues/898

**Comment by kaijli** [comment]
Need values for the following inputs:

```
        String db_dir                # this is most important one that I don't think I can find
        String container_run_cmd     # I'll see if I can find something in the repo
        Int threads = 1              # this is listed as 64 threads in the example 
```

I might not need to use the container_run_cmd if i follow this script https://code.jgi.doe.gov/img/img-pipelines/img-annotation-pipeline/-/blob/5.3/bin/annotate.sh?ref_type=heads#L171-174

Upon basic testing, if the `img-omics` image that we use in mg_annotation is updated to contain the genomad script, it would be much nicer than having to call the container_run_cmd

**Comment by kaijli** [comment]
Figured out container issue, working on getting a container that jaws can see and have a test run started

**Comment by ssarrafan** [comment]
Is this still actively being worked on @kaijli ? I don't see any updates in 2 weeks so will remove from board and add the backlog label unless you let me know it's active.

FYI @aclum

**Comment by aclum** [comment]
Actively in progress, we need to debug some permission issues.

**Comment by ssarrafan** [comment]
@aclum @kaijli still actively being worked on?

**Comment by aclum** [comment]
Yes, we have a new docker image from Marcel which was built and pushed to dockerhub today and @kaijli  is running tests now.

**Comment by aclum** [comment]
this is active, moving to the next sprint.

**Comment by ssarrafan** [comment]
@kaijli will leave this on this board since you're working this week. If you need help moving it to the next sprint ping me. Happy holidays.

**Comment by kaijli** [comment]
successful run at jaws id 98507

**Comment by kaijli** [comment]
PR opened, responding to feedback

**Comment by kaijli** [comment]
moving genomad and updating docker container

**Comment by ssarrafan** [comment]
No updates for 2 weeks. Removing from sprint. Adding backlog label.

**Comment by ssarrafan** [comment]
Removed from backlog, assuming this is active? 
@aclum can you review https://github.com/microbiomedata/mg_annotation/pull/44

**Comment by aclum** [comment]
review is complete, there are a few very small number of changes let.
